src/vellum/workflows/nodes/core/try_node/node.py

```diff
@@ -70,15 +73,20 @@ class Outputs(BaseNode.Outputs):
     def run(self) -> Iterator[BaseOutput]:
         subworkflow = self.subworkflow(
             parent_state=self.state,
-            context=self._context,
+            context=WorkflowContext(
+                _vellum_client=self._context._vellum_client,
+            ),
+        )
+        subworkflow_stream = subworkflow.stream(
+            event_filter=all_workflow_event_filter,
         )
-        subworkflow_stream = subworkflow.stream()
 
         outputs: Optional[BaseOutputs] = None
         exception: Optional[NodeException] = None
         fulfilled_output_names: Set[str] = set()
 
         for event in subworkflow_stream:
+            self._context._emit_subworkflow_event(event)
```

Comment:
This is the key piece that emits events from this subworkflow back up to the parent.

Could be controversial, since it means that nodes always has access to the main workflow event emitter and could emit whatever it wants, but hopefully the underscore'd private attributes are enough to say "don't do that"